go/oasis-node/cmd/node/node.go

```diff
@@ -570,8 +542,18 @@ func newNode(testNode bool) (*Node, error) {
 		}
 		node.svcMgr.Register(node.svcTmnt)
 		node.Epochtime = node.svcTmnt.EpochTime()
-
-		// Initialize the various node backends.
+		node.Beacon = node.svcTmnt.Beacon()
+		node.KeyManager = node.svcTmnt.KeyManager()
+		node.Registry = node.svcTmnt.Registry()
+		node.svcMgr.RegisterCleanupOnly(node.Registry, "registry backend")
```

Comment:
This doesn't seem right? The tendermint service should dispatch cleanup to its own backends?

Reply:
added a service manager in the tendermint service and this is done there now. Changes are in a separate commit for easier review (will fixup the "cr fixes" commit before merging)